backend/Alpaki.Tests.IntegrationTests/InvitationsTest/InviteAVolunteerTests.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Net.Http;
+using System.Net.Mail;
+using System.Threading.Tasks;
+using Alpaki.CrossCutting.Enums;
+using Alpaki.Logic.Features.Invitations.InviteAVolunteer;
+using AutoFixture;
+using FluentAssertions;
+using Xunit;
+
+namespace Alpaki.Tests.IntegrationTests.InvitationsTest
+{
+    public class InviteAVolunteerFake
+    {
+        public string Email { get; set; }
+    }
+
+    public class InvitationResponse
+    {
+        public InvitationItem[] Invitations { get; set; }
+        public class InvitationItem
+        {
+            public int InvitationId { get; set; }
+            public string Email { get; set; }
+            public string Code { get; set; }
+            public InvitationStateEnum Status { get; set; }
+        }
+    }
+    public class InviteAVolunteerTests : IClassFixture<CustomWebApplicationFactory>
+    {
+        private readonly HttpClient _client;
+        private readonly GraphQLClient _graphQL;
+        private readonly Fixture _fixture;
+
+        public InviteAVolunteerTests(CustomWebApplicationFactory factory)
+        {
+            _client = factory.CreateClient();
+            _graphQL = new GraphQLClient(_client);
+            _fixture = new Fixture();
+        }
+
+        [Fact]
+        public async Task Basic_scenario()
+        {
+            var (fake,request )= _fixture
+                .Build<InviteAVolunteerFake>()
+                .With(
+                    x => x.Email,
+                    _fixture.Create<MailAddress>().Address
+                )
+                .Create()
+                .WithJsonContent();
+
+            var query = @"
+query {
+  invitations {
+    invitationId
+    email
+    code
+    status
+  }
+}
+";
+            var response = await _client.PostAsync("/api/invitations", request);
```

Comment:
w sumie to są 3 testy w jednym :D

1. robi tylko posta i potwierdza żę jest OK. taki bardziej smoke!
2. dodaje kilka zaproszeń i robi query 
3. może sprawdzać czy mediatR zrobił następnego publisha. za dużo upchałeś w jednym

Reply:
A nie powinno to być tak, że jak są testy integracyjne to może być więcej asercji, ponieważ są czasowo kosztowne a w taki sposób oszczędzimy czas potrzebny na setup testu(u nas to aż takiego znaczenia nie ma akurat, bo dużo ich narazie nie mamy), kosztem tego, że teraz nie ma tylko jednego powodu dlaczego test nie przeszedł.